### ISSUE TITLE
everything but glide dosent work

### ISSUE THREAD

**Issue by MinoaBaccus** [HackerHansen/1v1.YEET-v2#36]
pls fix

**Comment by HackerHansen** [comment]
Which browser are you using, and which version of Cheat Engine are you on?

**Comment by MinoaBaccus** [comment]
im using Firefox and i downloaded cheat engine yesterday the hacks worked last night

**Comment by MinoaBaccus** [comment]
if u want i can get in a discord call wiht you and show u

**Comment by MinoaBaccus** [comment]
also speed hack dident even work lastnight i think its broken entirly

**Comment by MinoaBaccus** [comment]
now everything exept glyde is working

**Comment by HackerHansen** [comment]
Thanks for letting me know, I'll fix it later if I'm bored or something lol

**Comment by MinoaBaccus** [comment]
alright thanks btw you shuld make hacks for justfall.lol its made byu the same ppl at 1v1.lol

**Comment by MinoaBaccus** [comment]
also i found a kid wiht speed in 1v1.lol just now

**Comment by MinoaBaccus** [comment]
i think it works now
but everyonce and a while when trying to attach it crashes

**Comment by MinoaBaccus** [comment]
also speed still dosent worlk for me and my friends

**Comment by HackerHansen** [comment]
Speed is broken, it's the built-in cheat engine speedhack thingy 

I might get rid of the speed slider later

**Comment by sunnythedog2020** [comment]
how do you activate rapid fire

**Comment by sunnythedog2020** [comment]
@HackerHansen pls answer

**Comment by HackerHansen** [comment]
> how do you activate rapid fire

I think it's broken rn, I'm too lazy to check. Maybe I'll fix it later but idk when at this point, I might just archive the repo tbh

**Comment by sunnythedog2020** [comment]
is there a keybind?

**Comment by sunnythedog2020** [comment]
@HackerHansen

**Comment by HackerHansen** [comment]
> is there a keybind?

no. 

Also can you please read my previous comment...

**Comment by sunnythedog2020** [comment]
can you please fix it please im beging beging you

**Comment by HackerHansen** [comment]
> can you please fix it please im beging beging you

Let me rephrase my comment: 

At the present moment, the cheat commonly referred to as "rapid fire" included in the trainer in this repository is likely not functioning as intended. According to current circumstances, one may draw the conclusion that I am in a state known as "lazy" and do not want to check or fix this issue, but I may randomly decide to do so in the future. However, I am not sure when or if this will ever happen, and if I decide that I no longer want to work on this project I will perform an action known as "archiving a repository"

**Comment by HackerHansen** [comment]
Fixed, closing
